### ISSUE TITLE
What exactly does a "theme" encompass?

### ISSUE THREAD

**Issue by insanityfarm** [bnb/awesome-hyperterm#38]
Is there consensus on the minimum modifications a package must include to be a "theme," or what modifications are major enough that they reach beyond what a theme should do?

I just published my first HyperTerm plugin: [`hyperterm-maclite`](https://www.npmjs.com/package/hyperterm-maclite). I'm calling it a theme because it makes mostly aesthetic changes to tab appearance, including moving tab close buttons to the left. But I just saw the plugin `hyperterm-close-on-left` listed here, which does that same thing in a package by itself. Is my plugin too ambitious in that respect to be called a theme? On the other hand, the plugin doesn't make any changes to fonts or offer much other theming _inside_ the terminal window. Have I miscategorized it?

I'm considering removing my plugin's color overrides and rebranding it as a "window theme" (or something to that effect) that would be compatible with other themes listed here. No reason you shouldn't be able to mix-and-match my window styles with other people's color themes. Sorry if this is the wrong place to discuss this, I'm hoping to get some input from other HyperTerm plugin devs.

**Comment by bnb** [comment]
I definitely think it's a theme. You're doing a lot with it, which isn't bad - what I'd do is offer two things: the compilation of all the features, and then each feature individually as a separate package. 

I'd categorize themes as significant visual enhancements, and that they're one-at-a-time. You can have multiple plugins, but multiple themes would clash and error.

**Comment by insanityfarm** [comment]
@bnb Thanks for the input. I went ahead and unpublished the `hyperterm-maclite` theme, removed the non-tab styles from my code, and republished as the [`hyperterm-mactabs`](https://www.npmjs.com/package/hyperterm-mactabs) plugin (not theme). I have updated pull request #37 to list this under Plugins instead of Themes. Hope that's okay!

I may release some themes of my own (looks like we desperately need some light ones!) but I want to make sure this tab enhancement remains separate and compatible with as many third-party themes as possible.
